apps/condo/k6/src/registerBillingReceiptFile.test.ts

```diff
@@ -0,0 +1,184 @@
+import { faker } from '@faker-js/faker/locale/ru'
+import dayjs from 'dayjs'
+import { check } from 'k6'
+import crypto from 'k6/crypto'
+import encoding from 'k6/encoding'
+
+import {
+    setupCondoAuth,
+    createOrganization,
+    createBillingIntegration,
+    createBillingIntegrationOrganizationContext,
+    sendAuthorizedRequest,
+    registerBillingReceiptFile,
+} from './utils'
+
+const RECEIPT_PER_ACCOUNT_NUMBER = 50
+const TOTAL_RECEIPT_NUMBER = 100
+const DURATION = '60s'
+
+export const options = {
+    tags: { testid: 'registerBillingReceiptFile', serverUrl: __ENV.BASE_URL },
+    scenarios: {
+        registerBillingReceiptFile: {
+            exec: 'registerBillingReceiptFileCase',
+            executor: 'constant-vus',
+            duration: DURATION,
+            vus: 1,
+        },
+    },
+    thresholds: {
+        http_req_failed: ['rate<0.01'],
+        http_req_duration: ['med<14000'],
+    },
+}
+
+export function setup () {
+    const { token } = setupCondoAuth(true)
+
+    const createdOrganization = createOrganization({ token })
+    const organizationId = createdOrganization.json('data.obj.id')
+    const billingIntegration = createBillingIntegration({ token })
+
+    const billingContext = createBillingIntegrationOrganizationContext(
+        { token },
+        { id: organizationId },
+        billingIntegration.json('data.obj'),
+        { status: 'Finished' },
+    )
+    const data = {
+        token,
+        organizationId,
+        billingContext: billingContext.json('data.obj'),
+    }
+
+    const { receipts } = registerBillingReceipts(data)
+
+    const fileSize = 1024 * 1024 * 10 // 1024 bytes (kb) * 1024 => mb * 10 => 10mb
+    const bytes = crypto.randomBytes(fileSize)
+    const base64EncodedPDF = encoding.b64encode(bytes)
+
+    return {
+        ...data,
+        receipts,
+        base64EncodedPDF,
+    }
+}
+
+const randomNumber = (numDigits) => {
+    const min = 10 ** (numDigits - 1)
+    const max = 10 ** numDigits - 1
+    return faker.datatype.number({ min, max })
+}
+
+const createAddressWithUnit = () => __ENV.BASE_URL === 'http://localhost:3000'
+    ? `${faker.address.cityName()} ${faker.address.streetAddress(true)}`
+    : 'г Нижний Новгород, пр-кт Ленина, д 88 к 78, кв 1'
```

Comment:
A bit strange for OSS part...Can it moved to const / env?

Reply:
Let's keep it as is. I'm gonna to make a refactoring for load tests in separate PR, just to not block releases